X/xrt/build_tarballs.jl

```diff
@@ -0,0 +1,104 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message
+using BinaryBuilder
+
+name = "xrt"
+version = v"2.17"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/Xilinx/XRT.git", "a75e9843c875bac0f52d34a1763e39e16fb3c9a7"),
+    GitSource("https://github.com/Tencent/rapidjson.git", "ab1842a2dae061284c0a62dca1cc6d5e7e37e346"),
```

Comment:
Why this isn't a separate package?

Reply:
See #9272. As soon as this gets merged, I could add it as dependency.